docs(observability_pipelines): add WebSocket source page (#37197)
DataDog/documentation: layouts/shortcodes/observability_pipelines/prerequisites/websocket.en.md

> To use Observability Pipelines' WebSocket source, you need the following information available:

Reviewer: ```suggestion
To use Observability Pipelines' WebSocket source, you must have the following information:
```

> 1. The full WebSocket URI that the Observability Pipelines Worker connects to. For example, `wss://example.com/stream`.

Reviewer: ```suggestion
- The full WebSocket URI that the Observability Pipelines Worker connects to, such as `wss://example.com/stream`.
```

> 2. If the endpoint requires authentication, the credentials for your chosen authorization strategy (a username and password, a bearer token, or a custom `Authorization` header value).

Reviewer: ```suggestion
- If the endpoint requires authentication, the credentials for your chosen authorization strategy (a username and password, a bearer token, or a custom `Authorization` header value).
```

> The WebSocket source connects the Observability Pipelines Worker as a client to your upstream WebSocket endpoint and ingests the messages it receives as logs. The endpoint must accept WebSocket connections at the URI that you set as an environment variable when you install the Worker.

Reviewer: I don't think we need this paragraph here. It restates what was mentioned in the Overview. If we want the second sentence, it should probably go where the user enters in the URI (step 1 of setup).
```suggestion
```
